### ISSUE TITLE
Fill shape

### ISSUE THREAD

**Issue by FPGA-KING** [sameer/svg2gcode#15]
Hello I want to fill the shape.

Is it possible to draw filled shape?

For example If I draw star image as like example 

I want the library to make automatically tool path for fill the star.

**Comment by sameer** [comment]
Hi there,

svg2gcode does not support fills at the moment ([see FAQ](https://github.com/sameer/svg2gcode#faq--interesting-details)).

What kind of fill are you looking to do? Definitely open to collaborating on this :slightly_smiling_face: 

Thanks,
Sameer

**Comment by FPGA-KING** [comment]
Raster path

**Comment by sameer** [comment]
That seems like a reasonable use case. Is this something you'd be interested in making a pull request for?

**Comment by FPGA-KING** [comment]
> That seems like a reasonable use case. Is this something you'd be interested in making a pull request for?

Yes, It is. I have idea for it. but the problem is that I can use CPP only. 

I will first convert it to cpp code

**Comment by sameer** [comment]
Cool, this is very exciting. Happy to assist in the translation to Rust process.

**Comment by sameer** [comment]
>  the plotter raises the Z-axis and does not leave connection lines?

You can add in some g-code for that as the tool off sequence. 
![image](https://github.com/sameer/svg2gcode/assets/11097096/43b54742-c701-4139-b9b1-c7026090e12d)

Is this what you were looking for?

**Comment by chrisgwgreen** [comment]
Any developments on the fill feature?

**Comment by sameer** [comment]
We are getting closer to this: #118 adds filled polygon support internally, but I still feel there are some open questions on how that should work in practice.
